shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/common/util/ColumnExtractor.java

```diff
@@ -61,6 +62,25 @@ public static Collection<ColumnSegment> extract(final ExpressionSegment expressi
         return result;
     }
     
+    /**
+     * Check `IS NULL` or `IS NOT NULL` on right.
+     *
+     * @param expression BinaryOperationExpression
+     * @return true if `IS NULL` or `IS NOT NULL` on the right of BinaryOperationExpression.
+     */
+    public static boolean isNullOnTheRightOfExpressionSegment(final BinaryOperationExpression expression) {
```

Comment:
@cheese8 Since this class is called ColumnExtractor, its responsibility is to extract Columns, so I don't think isNullOnTheRightOfExpressionSegment should be placed in this class. 

In addition, for the judgment of whether the right side is NULL, whether we can do it in the SQL parsing stage, the judgment through the string is likely to fail due to case or space.

Reply:
I will try to resolve it with parse way.